casper/src/test/scala/coop/rchain/casper/genesis/contracts/PoSSpec.scala

```diff
@@ -15,7 +15,7 @@ class PoSSpec
     extends RhoSpec(
       CompiledRholangSource("PoSTest.rho", NormalizerEnv.Empty),
       Seq.empty,
-      120.seconds,
+      240.seconds,
```

Comment:
What did you determine was the source of this increase?

Reply:
`ListOps!take`.

Reply:
I guess that's another reason to make `take` a built-in.